### PR TITLE
INTLY-5013 rename installation to RHMI

### PR DESCRIPTION
**Pull request by pb82**
Rename `Installation` to `RHMI` and update all references

### CODE REVIEW

**Comment by KevFan** [comment]
Verified successfull install on OSD cluster 

/lgtm
/approve

File: pkg/apis/enmasse/v1beta1/zz_generated.openapi.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/integr8ly/integreatly-operator/pkg/apis/enmasse/v1beta1.BrokeredInfraConfig": schema_pkg_apis_enmasse_v1beta1_BrokeredInfraConfig(ref),
-		"github.com/integr8ly/integreatly-operator/pkg/apis/enmasse/v1beta1.StandardInfraConfig": schema_pkg_apis_enmasse_v1beta1_StandardInfraConfig(ref),
+		"./pkg/apis/enmasse/v1beta1.BrokeredInfraConfig": schema_pkg_apis_enmasse_v1beta1_BrokeredInfraConfig(ref),
```
Comment (grdryn) [inline pkg/apis/enmasse/v1beta1/zz_generated.openapi.go]: @pb82 I notice that something in your toolchain has changed these from being fully-qualified to being locally sourced. Do you know what that is? What version of the operator-sdk are you using?

Note that this part of your change will be reverted by https://github.com/integr8ly/integreatly-operator/pull/354 at least (or maybe a different PR, if one gets merged before that one).

**Pull request merged**